subsys/net/lib/lwm2m/lwm2m_obj_server.c

```diff
@@ -33,8 +39,10 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #define SERVER_STORE_NOTIFY_ID		6
 #define SERVER_TRANSPORT_BINDING_ID	7
 #define SERVER_REG_UPDATE_TRIGGER_ID	8
+#if defined(CONFIG_LWM2M_RD_CLIENT_SUPPORT_BOOTSTRAP)
+#define SERVER_BOOTSTRAP_UPDATE_TRIGGER_ID	9
```

Comment:
Also, there seems to be much more resources added in object version 1.1, if we claim to supprot this version we should add them all to the object model.